documentation/airport-format.md

```diff
@@ -1,309 +1,223 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+* [NOTICE](## NOTICE)
+* [Template](## Template)
+* [Comments](## Comments)
+    * [Property Descriptions](### Property Descriptions)
+        * [Latitude, Longitude, Elevation](#### Latitude, Longitude, Elevation)
+        * [ICAO and IATA identifiers](#### ICAO and IATA identifiers)
+
+## NOTICE
```

Comment:
this heading can be removed.  lets leave this paragraph as it is, just a general description.